pkg/model/components/awsebscsidriver.go

```diff
@@ -31,23 +31,16 @@ var _ loader.OptionsBuilder = &AWSEBSCSIDriverOptionsBuilder{}
 
 func (b *AWSEBSCSIDriverOptionsBuilder) BuildOptions(o interface{}) error {
 	clusterSpec := o.(*kops.ClusterSpec)
-	cc := clusterSpec.CloudConfig
-	if cc.AWSEBSCSIDriver == nil {
-		cc.AWSEBSCSIDriver = &kops.AWSEBSCSIDriver{
-			Enabled: fi.Bool(false),
```

Comment:
When the driver is enabled by default, the code will stay like this, but flipping this bool.

Reply:
Sounds fair. Just more complicated than I expected.